cmake/Dependencies.common.cmake

```diff
@@ -311,7 +318,7 @@ if(BUILD_NVIMAGECODEC)
       set(DALI_INSTALL_REQUIRES_NVIMGCODEC "")
     else()
       message(STATUS "ARCH is set to $ENV{ARCH}")
-      set(NVIMGCODEC_PACKAGE_NAME "nvidia-nvimgcodec-cu${CUDA_VERSION_MAJOR}[all]")
+      set(NVIMGCODEC_PACKAGE_NAME "nvidia-nvimgcodec-cu12[all]")
```

Comment:
Why? Is nvimgcodec for CUDA 11 discontinued? Is future-proofing for CUDA13? Does nvimgcodec for CUDA 12 work with CUDA 11 DALI build?

Reply:
Leftover, fixed.